AlfrescoApp/Model/Managers/AnalyticsManager.m

```diff
@@ -284,6 +308,7 @@ - (void)startAnalyticsType:(AnalyticsType)type
             if (self.googleAnalyticsHasStarted == NO)
             {
                 [[GAI sharedInstance] trackerWithTrackingId:GA_API_KEY];
+                [[GAI sharedInstance] setDispatchInterval:1];
```

Comment:
Is this setting the dispatch interval to 1 second? Does it need to be so different from the 120 second default? If it's for testing purposes, perhaps only set it when the app is in DEBUG mode?